products/pluto/metadata.yml

```diff
@@ -0,0 +1,675 @@
+name: primary_land_use_tax_lot_output_pluto
```

Comment:
curious, is this the unique id in Socrata? If I understand correctly, command to generate metadata is 
```bash
python3 metadata.py export <four_four> <output_path>
```